File: src/scala11-sources-test/src/main/scala-jdk11/com/example/MyJavaVersionDependentImpl.scala
package com.example

import scala.Console

import java.io.File
import java.io.IOException

class MyJavaVersionDependentImpl {
  def print(): Unit = {
    println(Console.GREEN + "Class from `java11`, while running Java " + System.getProperty("java.version") + Console.RESET)
  }
}

File: src/sbt-test/plugin-tests/java11-sources-test/src/main/java-jdk11/com/example/MyJavaVersionDependentImpl.java
package com.example;

import scala.Console;

import java.io.File;
import java.io.IOException;

public class MyJavaVersionDependentImpl {
  public void print() throws IOException {
    System.out.println(
      Console.GREEN() +
        "Class from `java11`, while running Java " + System.getProperty("java.version") +
        Console.RESET());
  }
}
